src/engine/engine.cpp

```diff
@@ -37,6 +37,15 @@
 #endif
 #include <fmt/printf.h>
 
+
+#if defined(_WIN32) && defined(HAVE_GUI)
+  #include "../gui/gui.h"
```

Comment:
Including the GUI in the engine is a bad practice (and may hinder my future relicensing plans).